src/main/java/com/example/sixthmafiabot/Bot.java

```diff
@@ -36,7 +35,10 @@ public class Bot extends TelegramLongPollingBot {
     GameService gameService;
 
     @Autowired
-    EnvironmentRepository environmentService;
+    UserService userService;
+
+    @Autowired
+    SpringEnvironmentRepository environmentService;
```

Comment:
wtf? SpringEnvRepo - environmentService
fix this shit

Reply:
+